panda/src/bullet/bulletRigidBodyNode.cxx

```diff
@@ -47,6 +47,28 @@ BulletRigidBodyNode(const char *name) : BulletBodyNode(name) {
 }
 
 /**
+ * Do not call the copy constructor directly; instead, use make_copy() or
+ * copy_subgraph() to make a copy of a node.
+ */
+BulletRigidBodyNode::
+BulletRigidBodyNode(const BulletRigidBodyNode &copy) :
+  BulletBodyNode(copy)
+{
+  _motion = new MotionState(*copy._motion);
```

Comment:
Since the rigid body is initialised with the motion state, you need to call setMotionState to initialise it with the new motion state.